content/S2016/S16-event-office-move.md

```diff
@@ -0,0 +1,11 @@
+Title: Our office has moved!
+Date: 2016-8-15
+Category: Events
```

Comment:
Yeah I think you should move it to a blog. The blog will show up as the thing on the front page until we make a new blog/event post.